src/node-explorer-provider.ts

```diff
@@ -33,74 +33,6 @@ export class NodeExplorerProvider
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public onDidChangeTreeData: vscode.Event<any> = this._onDidChangeTreeData.event;
 
-  refreshAll() {
```

Comment:
All this didn't change; I just moved it further down the file after the constructor.